assets/src/dashboard/constants.js

```diff
@@ -51,6 +56,8 @@ export const Z_INDEX = {
   POPOVER_PANEL: 10,
 };
 
+export const PAGE_RATIO = PAGE_HEIGHT / PAGE_WIDTH;
```

Comment:
A nit, but an important one. We committed everywhere to always have `ratio = width / height`. Otherwise it's too easy to forget/confuse.

Reply:
Ah okay. I can flip this and then switch the mult to division where we use it!

Reply:
Yeah. Otherwise approved. Feel free to merge as soon as the ratio is flipped.